docs/gcloud-auth.rst

```diff
@@ -58,15 +58,68 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful if your
+code needs to run in many different environments or if you just don't
+want authentication to be a focus in your code.
+
+However, if your code will only run in one place, you may want to
+use an explicit set of credentials suited to your environment. Instead, you
+may want to use two separate accounts to simultaneously access data from
+different projects.
+
+After creating a
+:class:`Credentials <oauth2client.client.Credentials>` object specific to
+your environment, you can pass it directly to a
+:class:`Client <gcloud.client.Client>`:
+
+.. code:: python
+
+    client = Client(credentials=credentials)
+
+.. _oauth2client: http://oauth2client.readthedocs.org/en/latest/
```

Comment:
Yea, didn't see that in this section though -- the link is used *after* the definition. Shouldn't we put these either at the end, or after the relevant section?>